network/src/protocols/discovery/mod.rs

```diff
@@ -378,6 +373,22 @@ where
     }
 }
 
+/// The note which has been verified its validity
+struct VerifiedNote {
+    peer_id: PeerId,
+    peer_info: VerifiedPeerInfo,
+    /// the raw `Note` sent from remote
+    raw: Note,
```

Comment:
How do we feel about converting from protobuf to LCS so we don’t have to keep around the raw note? :)

Reply:
Thank you for your review! I apologize for my lack of ability but please let me know what the `LCS` means. 🙏💦

Reply:
Do you mean this? 💡 
https://github.com/libra/libra/tree/master/common/lcs

Reply:
Sorry that comment was more meant for @bothra90 and does not impact this PR, sorry for the confusion :)

Reply:
Oh I see. 😃 The conversion from protobuf to LCS is also interesting. ✨